openstack_controller/tests/common.py

```diff
@@ -286,7 +286,7 @@
         "name": "newly_added_server",
         "status": "ACTIVE",
         "tags": [],
-        "tenant_id": "6f70656e737461636b20342065766572",
+        "tenant_id": "6f70656e737461636b20342065766573",
```

Comment:
What was this change for?

Reply:
It looked like a copy/past. In reality, I believe tenant_id should be unique.

Reply:
Nevermind, Good catch, my tests were wrong. I fixed them